homeassistant/components/http.py

```diff
@@ -202,6 +202,11 @@ def register_path(self, method, url, callback, require_auth=True):
         """ Regitsters a path wit the server. """
         self.paths.append((method, url, callback, require_auth))
 
+    def log_message(self, fmt, *args):
```

Comment:
I don't get why these are here. It would be highly preferred if we don't have to spread the fact that we route our log to a file throughout the app.